### DISCUSSION TITLE
Next release and schedule

### DISCUSSION THREAD

**Discussion by oharapaGitHub** [pytorch/torchtune#1579] [General]
I see a good amount of functionality, quality of life features, and refactoring has go into torchtune since the last release.  Has a date been set for the next release?

**Comment by joecummings** [comment]
Hey @oharapaGitHub - we actually just cut the RC today for v0.3.0! Aiming for a Monday release

**Comment by oharapaGitHub** [comment]
Excellent, looking forward to it
